config.json

```diff
@@ -536,7 +536,13 @@
       "core": false,
       "difficulty": 6,
       "slug": "spiral-matrix",
-      "topics": null,
+      "topics": [
+        "loops",
+        "arrays",
+        "numbers",
+        "mathematics",
+        "logic"
```

Comment:
Great suggestions! :) I agree with `arrays`, `mathematics` and `loops`. I'm not so sure about `numbers` and `logic`. `numbers` isn't a topic listed in the [topics list](https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt), `integers` would probably be a better topic. `logic` is used in this exercise, but you could argue that about every exercise. I wouldn't say logic is very important in this exercise compared to other exercises so I would be tempted to leave that topic out.

I would also add `matrices` as a topic :)

Reply:
Hi @FridaTveit , yes i agree with your suggestion. I've changed it